docs/src/staking/stake-accounts.md

```diff
@@ -76,8 +76,21 @@ with token balances of your choosing.
 The same stake and withdraw authorities can be assigned to multiple
 stake accounts.
 
-Two stake accounts that are not delegated and that have the same authorities
-and lockup can be merged into a single resulting stake account.
+#### Merging stake accounts
+Two stake accounts that have the same authorities and lockup can be merged into
+a single resulting stake account. A merge is possible between two stakes in the
+following states with no additional conditions:
+
+* two fully deactivated stakes
+* an inactive stake into a fully activating stake
+
+For the following cases, the voter pubkey and vote credits observed must match:
+
+* two fully activating stakes
```

Comment:
Ditto

Reply:
Here would you say "two stakes with the same activating epoch"?

Reply:
Do you think, "two activating stakes during their activation epoch," would clearly imply that the activation epochs are the same?  It kinda starts to get wordy if you make it more explicit :thinking:

Reply:
Hm, I'm not sure it's clear unfortunately -- "during their activation epoch" to me just sounds like we mistakenly made it redundant.  How about "two activating stakes with the same activation epoch"?

Reply:
> How about "two activating stakes with the same activation epoch"?

Unfortunately that's inaccurate in the case that they share an activation epoch `E`, but in epoch `E + N` (`N>0`) , have not yet fully activated (no partially active stake account can ever be merged).  We may have to relent and say "activation epoch" twice...  "two activating accounts that share an activation epoch, during the activation epoch" ?

Reply:
fair enough, clarity ftw!